src/App.java

```diff
@@ -42,7 +42,7 @@ of this software and associated documentation files (the "Software"), to deal
 
 public class App implements Constants {
 	
-	public static final String ver = "r3.2";
+	public static final String ver = "r3.3";
```

Comment:
According to best minecraft practices major version is locked to **1**.
According to best symdev practices, we have **4.3** as version but **3.3** as release. If all versions are releaseable (every unique version is a public release), is it better to synchronize them (and drop version/release splitting at all)?

Reply:
The first release version was 1.3.1, so...